src/demo/SidebarSections.tsx

```diff
@@ -1,5 +1,5 @@
 import { h, FunctionComponent } from 'preact'
-import { useCallback, useRef } from 'preact/hooks'
+import { useCallback, useRef } from 'preact/compat'
```

Comment:
Why this change?

Reply:
Yeah just a refactor to align with hooks imports from other places. Preact said [there're 2 ways to import hooks](https://preactjs.com/guide/v10/hooks), and I think `preact/compat` is more generic here.